### PR TITLE
Fixed Few Lint Errors

### LINKED ISSUE #6: Fix code lint erros to increase code quality
Increase code quality by fixing the following lint errors:
- [x] >index.html
|  1| <!doctype·html>
|    | [MAJOR] HTMLLintBear:
|    | Document Type: Use HTML5: Change "<!doctype html>" to "`<!DOCTYPE html>`".

- [x] >i|   2| ····<script··src="js/tiles.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | Extra whitespace: Use whitespaces only where expected and be consistent: Remove the extra whitespaces.

- [x] >i|   3| ····<script··src="js/levels.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for script tags is "text/javascript" so it can be safely omitted: Remove the type attribute from the script tag.

- [x] >iindex.html
|   4| ····<script··src="js/player.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | Extra whitespace: Use whitespaces only where expected and be consistent: Remove the extra whitespaces.

- [x] >iindex.html
|   7| ····<script··src="js/player.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for script tags is "text/javascript" so it can be safely omitted: Remove the type attribute from the script tag.

- [x] >iindex.html
|   8| ····<script··src="js/gui.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | Extra whitespace: Use whitespaces only where expected and be consistent: Remove the extra whitespaces.

- [x] >iindex.html
|   8| ····<script··src="js/gui.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for script tags is "text/javascript" so it can be safely omitted: Remove the type attribute from the script tag.

- [x] >iindex.html
|   9| ····<script··src="js/level.js"·type="text/javascript"></script>
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for script tags is "text/javascript" so it can be safely omitted: Remove the type attribute from the script tag.

- [x] >ijs/gui.js
|  18| ······player.moveLeft()
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >ijs/gui.js
|  19| ····};
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [x] >ijs/gui.js
|  24| ······player.moveRight()
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >ijs/gui.js
|  25| ····};
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [x] >ijs/gui.js
|  30| ······player.moveUp()
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >ijs/gui.js
|  31| ····};
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [x] >ijs/gui.js
|  36| ······player.moveDown()
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >ijs/gui.js
|  37| ····};
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [x] >ijs/gui.js
|  43| }
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >ijs/gui.js
|   9| ··level·=·createTestLevel();
|    | [NORMAL] JSHintBear:
|    | 'createTestLevel' is not defined.

- [x] >ijs/gui.js
|  11| ··player·=·newPlayer();
|    | [NORMAL] JSHintBear:
|    | 'newPlayer' is not defined.

- [x] >ijs/level.js
|   2| const·tileWidth·=·429.544;·//·px,·real·width
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [x] >ijs/level.js
|   3| const·tileHeight·=·256.314;·//·px
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [x] >ijs/level.js
|  16| ····}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >js/level.js
|  23| ········const·position·=·{"x":·x,·"y":·y};
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [x] >js/level.js
|  24| ········const·pixelPosition·=·me.indexToPosition(position);
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [x] >js/level.js
|  39| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >js/level.js
|  42| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >js/level.js
|  45| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [x] >js/level.js
|  57| ······const·position·=·{"x":·x,·"y":·y};
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/level.js
|  61| ··if·(this.startTile·==·null)·{
|    | [NORMAL] JSHintBear:
|    | Expected '===' and instead saw '=='.

- [ ] >js/level.js
|  36| ······return·NullTile;
|    | [NORMAL] JSHintBear:
|    | 'NullTile' is not defined.

- [ ] >js/level.js
|   5| function·Level(specification)
|    | [NORMAL] JSHintBear:
|    | 'Level' is defined but never used.

- [ ] >js/player.js
|  43| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/player.js
|  46| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/player.js
|   7| ··this.currentTile·=·NullTile;
|    | [NORMAL] JSHintBear:
|    | 'NullTile' is not defined.

- [ ] >js/player.js
|   2| function·newPlayer()·{
|    | [NORMAL] JSHintBear:
|    | 'newPlayer' is defined but never used.

- [ ] >js/loading.js
|   6| ····if·(xmlHttp.readyState·==·4)·{
|    | [NORMAL] JSHintBear:
|    | Expected '===' and instead saw '=='.

- [ ] >js/loading.js
|   7| ······if·(xmlHttp.status·==·200)·{
|    | [NORMAL] JSHintBear:
|    | Expected '===' and instead saw '=='.

- [ ] >js/loading.js
|  13| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/loading.js
|   1| function·httpGetAsync(theUrl,·callback)
|    | [NORMAL] JSHintBear:
|    | 'httpGetAsync' is defined but never used.

- [ ] >js/tiles.js
|   2| const·NullTile·=·{
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
|   3| ··canEnterFromTheTop:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|   4| ··canEnterFromTheBottom:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|   5| ··canEnterFromTheRight:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|   6| ··canEnterFromTheLeft:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|   7| ··playerLeaves:·function(player){},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|   8| ··placeAtIn:·function(){return·this},
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/tiles.js
|  15| }
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/tiles.js
|  24| ··}
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/tiles.js
|  35| };
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [ ] >js/tiles.js
|  56| ImageCollection.prototype.playerEnters·=·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|  59| ImageCollection.prototype.playerLeaves·=·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
|  72| };
|    | [NORMAL] JSHintBear:
|    | Unnecessary semicolon.

- [ ] >js/tiles.js
| 102| const·OpenDoors·=·{
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 104| ··canEnterFromTheTop:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 105| ··canEnterFromTheBottom:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 106| ··canEnterFromTheRight:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 107| ··canEnterFromTheLeft:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 108| ··canLeaveToTheTop:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 109| ··canLeaveToTheBottom:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 110| ··canLeaveToTheRight:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js

| 111| ··canLeaveToTheLeft:·function(player)·{return·true;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js

| 112| ··viewFromTheTop:·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 115| ··viewFromTheLeft:·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 117| ··viewFromTheRight:·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 120| ··viewFromTheBottom:·function(player)·{
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 122| ··whenPlayerLeaves:·function(player){
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 124| ··whenPlayerEnters:·function(player){
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 147| const·door·=·{
|    | [NORMAL] JSHintBear:
|    | 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 150| ····canEnterFromTheTop:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 151| ····canLeaveToTheTop:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 159| ····canEnterFromTheRight()·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 160| ····canLeaveToTheRight()·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 168| ····canEnterFromTheRight()·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 169| ····canLeaveToTheRight()·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'concise methods' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

- [ ] >js/tiles.js
| 170| ····canEnterFromTheTop:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 171| ····canLeaveToTheTop:·function(player)·{return·false;},
|    | [NORMAL] JSHintBear:
|    | 'player' is defined but never used.

- [ ] >js/tiles.js
| 178| }
|    | [NORMAL] JSHintBear:
|    | Missing semicolon.

- [ ] >js/tiles.js
|   2| const·NullTile·=·{
|    | [NORMAL] JSHintBear:
|    | 'NullTile' is defined but never used.

- [ ] >js/tiles.js
|  17| function·PlayerStartsAt(tileSpecification)·{
|    | [NORMAL] JSHintBear:
|    | 'PlayerStartsAt' is defined but never used.

- [ ] >js/tiles.js
| 147| const·door·=·{
|    | [NORMAL] JSHintBear:
|    | 'door' is defined but never used.

- [ ] >js/levels.js
|   3| ··return·new·Level([
|    | [NORMAL] JSHintBear:
|    | 'Level' is not defined.

- [ ] >js/levels.js
|   4| ····[door.none,·door.right,·door.right,·door.right,·door.right,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   4| ····[door.none,·door.right,·door.right,·door.right,·door.right,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   4| ····[door.none,·door.right,·door.right,·door.right,·door.right,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   4| ····[door.none,·door.right,·door.right,·door.right,·door.right,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   4| ····[door.none,·door.right,·door.right,·door.right,·door.right,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   5| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   7| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   8| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   9| ····[door.none,·door.top,·door.both,·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'door' is not defined.

- [ ] >js/levels.js
|   6| ····[door.none,·door.top,·PlayerStartsAt(door.both),·door.both,·door.both,·door.top],
|    | [NORMAL] JSHintBear:
|    | 'PlayerStartsAt' is not defined.

- [ ] >js/levels.js
|  10| ····[NullTile,·door.none,·door.none,·door.none,·door.none,·door.none],
|    | [NORMAL] JSHintBear:
|    | 'NullTile' is not defined.

- [ ] >js/levels.js
|   1| function·createTestLevel()
|    | [NORMAL] JSHintBear:
|    | 'createTestLevel' is defined but never used.

(**Note:** A contributor is asked to fix only a about 7-10 lint errors and leave the rest for the other contributors as outlined in the GCI task)

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #6 
Fixed Few Lint Errors

### CODE REVIEW

**Comment by naveentherocker** [comment]
@niccokunzmann Review Pls

File: js/level.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function Level(specification)
   this.visit = function() {
     this.tiles.forEach(function(row){
       row.forEach(function(tile){
-        tile.visit();
+        tile.visit();z
```
Comment (harshit98) [inline js/level.js]: Remove 'z' from the end.

File: js/level.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ function Level(specification)
       return tileSpecification.placeAtIn(position, me);
     });
   });
-  if (this.startTile === null) {
+  if (this.startTile == null) {
```
Comment (harshit98) [inline js/level.js]: Can you tell me why did you removed other '=' sign ?

File: js/loading.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ function httpGetAsync(theUrl, callback)
   // from https://stackoverflow.com/a/4033310/1320237
   var xmlHttp = new XMLHttpRequest();
   xmlHttp.onreadystatechange = function() {
-    if (xmlHttp.readyState === 4) {
-      if (xmlHttp.status === 200) {
+    if (xmlHttp.readyState == 4) {
```
Comment (harshit98) [inline js/loading.js]: Why you're removing other `=` sign ?

**Comment by naveentherocker** [comment]
Closing and Opening a New One due to an error in Squashing

**Comment by naveentherocker** [comment]
image

**Pull request closed without merging**